editor/src/templates/editor-canvas.tsx

```diff
@@ -565,10 +565,6 @@ export class EditorCanvas extends React.Component<EditorCanvasProps> {
       this.props.dispatch([CanvasActions.setSelectionControlsVisibility(true)], 'canvas')
     }
 
-    if (this.props.model.focusedPanel !== 'canvas' && event.event !== 'WHEEL') {
-      return
-    }
```

Comment:
Does removing this mean that scrolling the navigator will now also scroll the canvas?

Reply:
I shall test that case.

Reply:
I can confirm that navigator and canvas scroll independently.

Reply:
Thanks for checking!